cli/src/stake.rs

```diff
@@ -846,16 +850,20 @@ pub fn process_split_stake(
     split_stake_account: &Keypair,
     split_stake_account_seed: &Option<String>,
     lamports: u64,
+    fee_payer: Option<&SigningAuthority>,
 ) -> ProcessResult {
+    let (fee_payer_pubkey, fee_payer) = fee_payer
+        .map(|f| (f.pubkey(), f.keypair()))
+        .unwrap_or((config.keypair.pubkey(), &config.keypair));
     check_unique_pubkeys(
-        (&config.keypair.pubkey(), "cli keypair".to_string()),
+        (&fee_payer_pubkey, "fee-payer keypair".to_string()),
```

Comment:
@mvines is the intention correct here?  The CLI keypair's role wasn't clear for this subcommand

Reply:
yep, lgtm